### PR TITLE
Fix path to css in main app's html template

### PR DESCRIPTION
**Pull request by amiroff**
:+1:

### CODE REVIEW

File: {{cookiecutter.project_name}}/server/apps/main/templates/main/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <link rel="stylesheet" href="{% static 'main/css/index.css' %}">
+    <link rel="stylesheet" href="{% static 'main_app/css/index.css' %}">
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/server/apps/main/templates/main/index.html]: Good catch! But, I prefer consistent naming. So, let's better rename `static/main_app` to `static/main`

Comment (amiroff) [reply {{cookiecutter.project_name}}/server/apps/main/templates/main/index.html]: Guess, @DmytroLitvinov is already on it. Thanks!

Comment (DmytroLitvinov) [reply {{cookiecutter.project_name}}/server/apps/main/templates/main/index.html]: Thank you for your attention :)

**Comment by sobolevn** [comment]
@amiroff thanks again 👍

**Pull request closed without merging**